### PR TITLE
translate to russian

### PR DESCRIPTION
**Pull request by paulmillr**
why not

looks more nice after the change

### CODE REVIEW

**Comment by RichardLitt** [comment]
PR on hold until we get another Russian in here. Too confusing at the moment.

File: test/basic.spec.js [hunk 1]
```diff
@@ -3,14 +3,14 @@ var writeGood = require('../write-good');
...
-      { index: 11, offset: 3, reason: '"few" is a weasel word' }
+      { index: 0, offset: 10, reason: '"Remarkably" есть слово горностай' },
+      { index: 11, offset: 3, reason: '"few" есть слово горностай' }
     ]);
   });
 
   it('should detect passive voice', function () {
     expect(writeGood('The script was killed')).toEqual([
-      { index: 11, offset: 10, reason: '"was killed" is passive voice' }
+      { index: 11, offset: 10, reason: '"was killed" есть пассивный звук' }
```
Comment (Termina1) [inline test/basic.spec.js]: '"was killed" это страдательный залог

File: test/basic.spec.js [hunk 2]
```diff
@@ -20,7 +20,7 @@ describe('writeGood', function () {
 
   it('should detect lexical illusions', function () {
     expect(writeGood('the the')).toEqual([
-      { index: 4, offset: 3, reason: '"the" is repeated' }
+      { index: 4, offset: 3, reason: '"the" есть повтор' }
```
Comment (Termina1) [inline test/basic.spec.js]: '"the" повторяется

File: test/basic.spec.js [hunk 3]
```diff
@@ -30,13 +30,13 @@ describe('writeGood', function () {
 
   it('should detect lexical illusions with line breaks', function () {
     expect(writeGood('the\nthe')).toEqual([
-      { index: 4, offset: 3, reason: '"the" is repeated' }
+      { index: 4, offset: 3, reason: '"the" есть повтор' }
```
Comment (Termina1) [inline test/basic.spec.js]: "the" повторяется

File: test/basic.spec.js [hunk 4]
```diff
@@ -30,13 +30,13 @@ describe('writeGood', function () {
...
     ]);
   });
 
   it('should detect lexical illusions with case insensitivity', function () {
     expect(writeGood('The the')).toEqual([
-      { index: 4, offset: 3, reason: '"the" is repeated' }
+      { index: 4, offset: 3, reason: '"the" есть повтор' }
```
Comment (Termina1) [inline test/basic.spec.js]: "the" повторяется

File: test/basic.spec.js [hunk 5]
```diff
@@ -46,7 +46,7 @@ describe('writeGood', function () {
 
   it('should detect sentences that start with "so"', function () {
     expect(writeGood('So the best thing to do is wait.')).toEqual([
-      { index: 0, offset: 2, reason: '"So" adds no meaning' }
+      { index: 0, offset: 2, reason: '"So" добавляет нет значения' }
```
Comment (Termina1) [inline test/basic.spec.js]: "So" не добавляет смысла

File: test/basic.spec.js [hunk 6]
```diff
@@ -64,14 +64,14 @@ describe('writeGood', function () {
 
   it('should detect clauses after a semicolon that start with "so"', function () {
     expect(writeGood('This is a test; so it should pass or fail.')).toEqual([
-      { index: 16, offset: 2, reason: '"so" adds no meaning' }
+      { index: 16, offset: 2, reason: '"so" добавляет нет значения' }
```
Comment (Termina1) [inline test/basic.spec.js]: "so" не добавляет смысла

File: test/basic.spec.js [hunk 7]
```diff
@@ -64,14 +64,14 @@ describe('writeGood', function () {
...
     ]);
   });
 
   it('should order suggestions by index', function () {
     expect(writeGood('It has been said that few developers write well.')).toEqual([
-      { index: 7, offset: 9, reason: '"been said" is passive voice' },
-      { index: 22, offset: 3, reason: '"few" is a weasel word' }
+      { index: 7, offset: 9, reason: '"been said" есть пассивный звук' },
```
Comment (Termina1) [inline test/basic.spec.js]: "been said" это страдательный залог

File: write-good.js [hunk 8]
```diff
@@ -1,8 +1,8 @@
 var checks = {
-  weasels  : { fn: require('weasel-words'),            explanation: 'is a weasel word' },
-  illusion : { fn: require('./lib/lexical-illusions'), explanation: 'is repeated' },
-  so       : { fn: require('./lib/starts-with-so'),    explanation: 'adds no meaning' },
-  passive  : { fn: require('passive-voice'),           explanation: 'is passive voice' }
+  weasels  : { fn: require('weasel-words'),            explanation: 'есть слово горностай' },
+  illusion : { fn: require('./lib/lexical-illusions'), explanation: 'есть повтор' },
```
Comment (Termina1) [inline write-good.js]: это повтор

File: write-good.js [hunk 9]
```diff
@@ -1,8 +1,8 @@
...
+  so       : { fn: require('./lib/starts-with-so'),    explanation: 'добавляет нет значения' },
```
Comment (Termina1) [inline write-good.js]: не добавляет смысла

File: write-good.js [hunk 10]
```diff
@@ -1,8 +1,8 @@
...
+  passive  : { fn: require('passive-voice'),           explanation: 'есть пассивный звук' }
```
Comment (Termina1) [inline write-good.js]: это страдательный залог

**Comment by Termina1** [comment]
weasel word, actually, have no analog in russian, the closest meaning is "пустое слово" or "расхожее выражение"

**Comment by RichardLitt** [comment]
Thanks @Termina1. Looks like the translation wasn't great before. @paulmillr want to update it?

**Comment by Termina1** [comment]
But, why you want to translate this to russian? What's the point?

**Comment by RichardLitt** [comment]
See @paulmillr - "why not".

**Comment by jackunion** [comment]
@Termina1 "слово-паразит"

**Comment by Termina1** [comment]
@jackunion no, it's not, "слово-паразит" is something like "so" or "well". Words that do not have meaning, but aimed to fill the gap between smth.

**Comment by paulmillr** [comment]
@Termina1 @jackunion 

circus bljad

**Comment by Termina1** [comment]
@paulmillr 
Блядский цирк

**Comment by jackunion** [comment]
@paulmillr я мимо проходил.

**Comment by RichardLitt** [comment]
I'm going to close this, as there has been no update in a long time and the conflicts have not been resolved. Feel free to reopen.

**Pull request closed without merging**